### PR TITLE
add: blog-post-section

### PR DESCRIPTION
**Pull request by prince314**

### CODE REVIEW

File: src/style/default.scss [hunk 1]
```diff
@@ -1331,23 +1332,25 @@ p {
...
 
-    & .blue {
-      & .up {
-        width: 270px;
+    .blue {
+      .up {
+        width: 280px;
         height: 325px;
         background-color: $lighter-blue;
-        margin: 0px 10px 0 10px;
```
Comment (happimaker) [inline src/style/default.scss]: You should code like this.
margin: 0 10px

File: src/style/default.scss [hunk 2]
```diff
@@ -1331,23 +1332,25 @@ p {
...
+        margin: 0px 8px 0 8px;
+
+
```
Comment (happimaker) [inline src/style/default.scss]: delete these empty lines

File: src/style/default.scss [hunk 3]
```diff
@@ -1408,37 +1439,37 @@ p {
         display: flex;
         flex-direction: row;
 
-        & .up {
-          width: 270px;
+        .up {
```
Comment (happimaker) [inline src/style/default.scss]: not a good coding style.

File: src/style/default.scss [hunk 4]
```diff
@@ -1477,41 +1508,612 @@ p {
...
+      margin: 0 8px 0 8px;
 
       @include white-plane-per;
 
       @include common-button (white, $green, 2px);
     }
 
-    & .red {
-      width: 270px;
+    .red {
```
Comment (happimaker) [inline src/style/default.scss]: coding style

File: src/index.html [hunk 5]
```diff
@@ -569,5 +569,136 @@ <h3>Achmad Doe</h3>
...
+          </div>
+          <div class="blog-post-content-email">
+            <div class="blog-post-content-email-left">
+              <div>
+                <button>ENTER YOUR EMAIL TO SUBSCRIBE NEWSLETTER</button>
+              </div>
+            </div>
+            <div class="blog-post-content-email-right">
+              <div>
+                <p>LOAD MORE BLOG POST</p><span class="fa fa-recycle"></span>
```
Comment (rogue707Dev) [inline src/index.html]: <span> tag is inline element so it's better to put it inside <p> tag

File: src/index.html [hunk 6]
```diff
@@ -569,5 +569,136 @@ <h3>Achmad Doe</h3>
         </div>
       </article>
     </section>
```
Comment (rogue707Dev) [inline src/index.html]: Put line break between section.

File: src/index.html [hunk 7]
```diff
@@ -569,5 +569,136 @@ <h3>Achmad Doe</h3>
...
+            </div>
+            <div class="blog-post-content-email-right">
+              <div>
+                <p>LOAD MORE BLOG POST</p><span class="fa fa-recycle"></span>
+              </div>
+            </div>
+          </div>
+        </article>
+      </div>
+    </section>
```
Comment (rogue707Dev) [inline src/index.html]: Put line break between section

File: src/style/default.scss [hunk 8]
```diff
@@ -94,7 +94,7 @@ p {
       align-items: center;
     }
 
-    & .text {
+     .text {
       font-family: 'Raleway';
       color: white;
```
Comment (rogue707Dev) [inline src/style/default.scss]: Remove space lines

File: src/style/default.scss [hunk 9]
```diff
@@ -283,20 +283,20 @@ p {
           padding: 10px 20px 10px 20px;
         }
 
-        & .desc {
+        .desc {
           font-size: 22px;
           color: white;
           font-weight: 600;
           padding-top: 20px;
           padding-bottom: 20px;
```
Comment (rogue707Dev) [inline src/style/default.scss]: It's better to make this as mixin

File: src/style/default.scss (hunk 3 above)
Comment (lazydeveloperme) [reply src/style/default.scss]: Please use more specific name rather than "up" to avoid class name conflict

File: src/index.html (hunk 5 above)
Comment (prince314) [reply src/index.html]: on my mind

**Pull request merged**